tests/runtime/call

```diff
@@ -246,6 +246,15 @@ ENV BPFTRACE_CACHE_USER_SYMBOLS=PER_PROGRAM
 PROG uprobe:./testprogs/uprobe_symres_exited_process:test { print(ustack); exit(); }
 EXPECT ^\s+test\+0\s+main\+[1-9][0-9]*
 AFTER ./testprogs/disable_aslr ./testprogs/uprobe_symres_exited_process
+ARCH x86_64
+TIMEOUT 5
+
+NAME ustack_elf_symtable
+ENV BPFTRACE_CACHE_USER_SYMBOLS=PER_PROGRAM
+PROG uprobe:./testprogs/uprobe_symres_exited_process:test { print(ustack); exit(); }
+EXPECT ^\s+test\+0\s+test2\+[1-9][0-9]*\s+main\+[1-9][0-9]*\s+0x[0-9a-fA-F]+\s+0x[0-9a-fA-F]+$
```

Comment:
I don't think that the part after `main\+[1-9][0-9]*` is necessary. We should only check that the stacktrace contains symbols from the original program.

Reply:
Done